src/d_main.c

```diff
@@ -2141,6 +2199,28 @@ void D_DoomMain(void)
             D_AddFile(myargv[p]);
     }
 
+  if (!M_CheckParm("-save"))
+  {
+    int i;
+    char *wadname = wadfiles[0], *oldsavegame = basesavegame;
+
+    for (i = mainwadfile; i < numwadfiles; i++)
+    {
+      if (FileContainsMaps(wadfiles[i]))
+      {
+        wadname = wadfiles[i];
+        break;
+      }
+    }
+
+    basesavegame = M_StringJoin(oldsavegame, DIR_SEPARATOR_S,
+                                M_BaseName(wadname), NULL);
+    free(oldsavegame);
+
+    NormalizeSlashes(basesavegame);
+    M_MakeDirectory(basesavegame);
```

Comment:
I think we should create a `saves/` directory, where we will put the subdirectories of the PWAD saves.

Reply:
Choco/Crispy use `savegames/`.

Reply:
I didn't know that, on Windows it saves in the executable directory. Surely `savegames/` is better.